codebase/superdarn/src.bin/tk/tool/combine_snd.1.0/makefile

```diff
@@ -0,0 +1,17 @@
+# Makefile for combine_snd
+# ========================
+# Author: E.G.Thomas
+# by E.G.Thomas
```

Comment:
this doesn't make sense :p 
copyright/disclaimer

Reply:
```suggestion
# Copyright (C) 2020, Dartmouth College
# Author: E.G.Thomas
# 
# This program is free software: you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.
# 
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU General Public License for more details.
# 
# You should have received a copy of the GNU General Public License
# along with this program.  If not, see <http://www.gnu.org/licenses/>.
```